src/assets/scripts/aircraft/AircraftFlightManagementSystem.js

```diff
@@ -72,12 +82,15 @@ export default class AircraftFlightManagementSystem {
             anything: false    // T/F flag for if anything is being "followed"
         };
 
+        // TODO: enumerate the magic numbers
         // set initial
         this.fp.altitude = clamp(1000, options.model.ceiling, 60000);
 
         if (options.aircraft.category === 'arrival') {
+            // TODO: why KDBG?
```

Comment:
correct. back in the good old days, "KDBG", aka "Debug Airport" was the first airport in the game. When needing to fill the origin and destination airports in, "KDBG" was an arbitrary choice so it wasn't empty.